.github/workflows/pypi_publish.yml

```
@@ -48,8 +56,104 @@ jobs:
           name: wheels-sdist
           path: ./wheelhouse/*.tar.gz
 
+  build-wheels:
+    name: 'Wheels: ${{ matrix.os }} ${{ matrix.arch }} ${{ matrix.python-version }}'
+    runs-on: ${{ matrix.os }}
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        python-version: ['3.12']
```

There should be a different wheel per python version, per OS, per arch
```suggestion
        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
```